pkg/shares/testdata/sample-block.json

```diff
@@ -0,0 +1 @@
nQGSh0BXQe2eYR5ucXoUMdEtebQq3vttEdwMnjmn8rofu1Q3bO+0aSQBVOgLO1k6/G0ZL/VsFBqAuRFcY20ha/i2P6sYZLE2nXTetQqx+VwD2ttgVsqh7FzLQ+/BjM5VnobVxLa8xTUNhaaK0WRBDOSOXR3MhFP/HK3+SpXCOGYyHg3BX+G6hzSX1D7rZZdH8k6ER+sCWf6izLm4+h2pkgVXQeyrcc5ZxeLCyq24C7jgIbRL1Zrwutah2sI1bFMKOzg5bnWPSqzX6rNzwOO1Ike6YPer88rop0CyWe/NNc5i4YutmxS5/cy7eVnPMc+kKzIPhWtKv7o/soc88c3QKPcdmTyeUat1refZ+v6dzjssvH/xvfb8GFpelYyhcGBEsWCwE6WdVIlU2SShLbd7ACY50TDw8f564QjA/D9xvUDQKFNfPM+2M1uzqvjT0sTIHv6fBOtMXYAE3cetqMz3L2NnAXVWgSVw/P7Z3QXNCgZWnLLJ5OPdY769RQMc16dRvAyIPxCCoYAw4wisSz0wLPUGrpH4s3uxMNfaHKBHBGmWy8lLxGWjCrSwnCAAl1suYTMCvSP2azNncsmHswJ7GzmtkLB8etyp1flVrpzYayZH/3EYKkCoVYcs2iHqQLo3Od/OyirhwbWauHuZfdxx1er9/Roo+DIayM9DrM9TgzEMygTnvt5GBS3yinCKAbr31CiCBsF9ICankdWMqKKHO1aDrIIzfhAjcdxx/wM1mORJLm5AZmIXN3tqGKcRDxiPv7mXEkm8b068WdtH5X9qXg9PBV/caUgpEhDAuGoYpmHJrYHfWCk5++EjSDUnvP7EWNkbKFBN7jdNCZkTgPSHJb2q0B+1mUkBYROlTDp2DqO+1Th9qySZIjlCP2gZaKUo4zCtxvap5hUmD4jgzpmjBEBuKBwSZlH/0fScXazO6PWxNve1eeUrdLUJ+Zrr8A9fpZbLXTTMdflX0EPrbNU+mUBRvMz54Mfc6VtrwYd9hKiBviO1yIDcYIJQcXG3J253KSjqpIv3Uff5i5XtZNcIY1LBEdp+r5UYpwSwv9zZC/IsDZb3kggp9A1XAZZsXcDK8bd3VHH6ZrdgIDZdyDUD+8XQ2AVVHF/j7vDD59Tcs0O9yz7+WQV2qv1DlskRtaRde6SLo62H3vYM6Au/VjRT0x5Fnz/SVEGQVnav6aWEAawlCOi8xaTXQwxxqsbG+/SpX47bwQ2wECoKIg/gvfe23sgOMHVJXVfw0eA7DzreMMAnfjWVE1Ap6vLpuTGtwcUCa0Hjfp031076q9GiKrbRP5Jj/NiVV0TmMfoJFm6PQctoLdHb39ZMpthcfIsknQIzn+7jtXxxkfSww3QIBIvnG3BLBeQba4uRER5wcgUOUc+mK+YKmvMmtWbDD0Tv3+5CDONxjlFbVWsR+LNNlEf/T+2dJb00Ol2tIOd069cmkPqmvH2eNjOv/W/bubA6Ogrq9cLrAP2HNySvLp2PN1QNvAnCrvnYnwqgmYewHMrIqXiF+atJHM2qgofidLBd2W+mUsrHzrnid7NsxHGvHz5oStqTSU8e2NhOqSUUu9Hfh7Bi8p0hfXj+zIPhZfFcJFHCKc1qDN6RhmYBkpihJ90/+m2lWY2+SUgLCjse3O6ohh3bgmozyItrIFSTu1KWrdiO5mJ/snHeVKX110ZDiHku/Jygq3j8dPSF5RfElfPJxeFgSbHWX2KGtAA5oEKhp4wnhZhnFEppZ1mFrXgdPO4AAwKZ3V/vwmfbv1xZ9zT7O5PU1DymWsdQqTtR4Bwi0t0/napqH534F0cEiKT4ZVvPErTGckYfEpAuTd1JbNekrTAT3Gdy9E97LKxBGAJiP7dcfRuZKaVtt62lSkOM9dK61YYY7TlRRsG31YV04Q4eoF1Zxh81xRyV19OjIjngHzYuG7KUnmt6uMc/aQjplIEfQXud7PYm8CP4DBaqd6cAgclmsW0XkLmHk04JTe7Vd6xaPAjhK+H/2Z59tiDAYOxUqybAPy/Un8p3kf9WfkqLVzQZviOJYoIEXsQBtAA3P1f3Cs0V8NZQZcZmHCL/cCCJ6V+V3sEmhUl1nblrTgBmpfdlZRcASB20XgFXT6LQDimSYv9ycrMwpJXRxHLEgnbCo72mrBh4p2opBbfZBLkhAnkqfV/mjS/sxAZ4kKmASHM/Wbqqia2mo6FIvk9OeyVwU1v15v5iGwDd77uGB95MfcA/7qXqgPrUxaZUl9fQ8R6kcqvWIXOu8+KbSm1IB58VRxnXYFFlcRoBr4lTWHPrBivtArOSIZ6fZ/HY9Dw9oj0lACaXxZEtkLl/vDNxPEx5B8MN+onhOzgLccK5q8RBGf8hA9ebClWvRmLvYT/+yTcvOmwPWgSs6nOYprckWc47tia2350CXU6EPCnzECSE/X3AcNnwgli2Z1T+vUbYrBr3jrUjmQpujFOsuJZV9oaAfXXPlN+Lsz4I7bil4kyomQ/w3q3gTQfWmmkU="}],"square_size":4,"hash":"uPWH3TVoZNb3NcyXN/vbX5N1mDZ8dfXMh9bhne04CpM="},"evidence":{"evidence":[]},"last_commit":{"height":3,"block_id":{"hash":"CnTXMWDvJXaHOjecWXb2nGvLsJ23GmgUbAYIZmteP2M=","part_set_header":{"total":1,"hash":"PgAxB6DYr8huhDZJWMnFSgYDCXwuF3jI1GNbnOskv9U="}},"signatures":[{"block_id_flag":2,"validator_address":"XbENo47oRQwd9YSP9vr+10SttTc=","timestamp":"2023-01-30T18:55:27.534092Z","signature":"CPafxD85F5yNxKP/9hm2ew4WLPRIo472AcVNMgOZh239V4AnKsRpL8FUwS14X48DpsZuzVCqLgUKY2+agupxBQ=="}]}}
```

Comment:
how was this testdata generated? how do we regenerate it? 

admittedly, I'm not the biggest fan of using test data because the tests that rely on this could pass indefinitely, even if something in the block changes. imo, we should at least have a function to regenerate it a la https://github.com/celestiaorg/celestia-node/blob/3a1353c0d122e64b991983cf889530e2c7747d10/state/integration_test.go#L120-L147 and even then we'd still want some CI or something to notify us that it changed :grimacing: 

if we want to avoid import cycles, then I would prefer that we have those tests in a test package, but this is just a preference and doesn't have to be done in this PR

Reply:
> how was this testdata generated? how do we regenerate it?

https://github.com/celestiaorg/celestia-node/blob/3a1353c0d122e64b991983cf889530e2c7747d10/state/integration_test.go#L120-L147

> how do we regenerate it?

I can add that to celestia-app